packages/main/src/themes/sap_horizon/Dialog-parameters.css

```diff
@@ -6,4 +6,5 @@
 	--_ui5_dialog_header_focus_left_offset: 1px;
 	--_ui5_dialog_header_focus_right_offset: 1px;
 	--_ui5_dialog_header_border_radius: var(--sapElement_BorderCornerRadius);
-}
+	--_ui5_dialog_block_layer_opacity: 0.2;
```

Comment:
This is the same as in "base" so no need to be duplicated. Please remove.

The "--_ui5_dialog_block_layer_opacity" should be 0.6 for Belize theme. Please add this override.